server.js

```diff
@@ -129,9 +130,13 @@ app.get('/api/oauth-redirect', (req, res) => { //redirects to the url configured
             apiModules.authentication.grantProjectPermissions({
                 contributor: contributorInfo.contributor.dataValues
             })
+            return contributorInfo
         })
-        .then(() => {
-            res.redirect(SITE_ROOT)
+        .then((contributorInfo) => {
+            if (contributorInfo.newUser) {
+                return res.redirect(`${SITE_ROOT}/onboarding`)
```

Comment:
Can we refactor this to an environment variable `OAUTH_REDIRECT_NEW_USERS` and `OAUTH_REDIRECT_EXISTING_USERS`

So it will be easier to maintain backwards compatibility with v1

Reply:
Resolved [d59c17f](https://github.com/setlife-network/trinary/pull/720/commits/d59c17f3d509f5034c9ca4919b9061be3623e993)